bioptim/dynamics/configure_problem.py

```diff
@@ -671,6 +690,12 @@ def define_cx(n_col: int) -> list:
                     combine_to=f"{name}_states" if as_states and combine_state_control_plot else combine_name,
                 )
 
+        if as_states_dot:
+            n_cx = nlp.ode_solver.polynomial_degree + 1 if isinstance(nlp.ode_solver, OdeSolver.COLLOCATION) else 2
+            cx = define_cx(n_col=n_cx)
+
+            nlp.states_dot.append(name, cx, mx_states_dot, nlp.variable_mappings[name])
+
```

Comment:
It seems that this implies COLLOCATION, please add a raise for other case if I am not mistaken. Otherwise, I suspect this will fail

Reply:
It won't fail if it's not COLLOCATION, it also works with IRK.